.github/workflows/autoupdate-prod.yaml

```diff
@@ -29,11 +35,14 @@ jobs:
       - name: Run docs generation
         if: steps.verify-changed-files.outputs.files_changed == 'true'
         run: make gen-docs
+      - name: Commit Generator Changes
+        run: |
+          git add . && git commit -m "fix: Generated SDK source code and docs"
       - name: Release updates
         if: steps.verify-changed-files.outputs.files_changed == 'true'
         working-directory: ./tools
-        env: 
-          GIT_BASE_REF: origin/main
+        env:
+          GIT_BASE_REF: ${{ github.sha }}
```

Comment:
I needed to make this change in order to test it from non main branch (like this PR)